pkg/ui/src/views/statements/barCharts.tsx

```diff
@@ -227,6 +228,40 @@ export function rowsBreakdown(s: StatementStatistics) {
   };
 }
 
+
+export function genericBarChart(s: NumericStat, count: number | Long) {
```

Comment:
I wish we had something more reusable around but I think this is a good idea for now. The other bar chart generation is too specialized.